rviz_common/include/rviz_common/message_filter_display.hpp

```diff
@@ -131,6 +131,7 @@ class MessageFilterDisplay : public _RosTopicDisplay
         std::bind(
           &MessageFilterDisplay<MessageType>::incomingMessage, this,
           std::placeholders::_1));
+      updateMessageQueueSize();
```

Comment:
Instead of updating the queue size after creating the filter, can we instead set the queue size directly on L128?